api/src/integration/java/org/pmiops/workbench/AbsorbAcceptanceTest.java

```diff
@@ -0,0 +1,57 @@
+package org.pmiops.workbench;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.pmiops.workbench.absorb.api.AuthenticateApi;
+import org.pmiops.workbench.absorb.api.EnrollmentsApi;
+import org.pmiops.workbench.absorb.api.UsersApi;
+import org.pmiops.workbench.absorb.model.AuthenticationRequest;
+import org.pmiops.workbench.google.CloudStorageClient;
+import org.pmiops.workbench.google.DirectoryServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+
+public class AbsorbAcceptanceTest extends BaseIntegrationTest {
+  @Autowired private CloudStorageClient cloudStorageClient;
+
+  @TestConfiguration
+  @ComponentScan(basePackageClasses = DirectoryServiceImpl.class)
```

Comment:
I'm still learning Spring so I'm unsure why this is needed. However, trying to use other classes / services resulted in either test breakages or additional configuration, so I've left this as-is.